SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -588,12 +617,7 @@ - (void)groupSend:(NSArray<SignalRecipient *> *)recipients
         if ([recipientId isEqualToString:[TSAccountManager localNumber]]) {
             continue;
         }
-        // We don't need to sent the message to all group members if
-        // it has a "single group recipient".
-        if (message.singleGroupRecipient && ![message.singleGroupRecipient isEqualToString:recipientId]) {
-            continue;
-        }
-        if ([message wasSentToRecipient:recipientId]) {
+        if (![message.sendingRecipientIds containsObject:recipientId]) {
```

Comment:
You're right.  In fact, this is redundant since we're consulting `message.sendingRecipientIds ` in `sendMessageToService` right before we call this method.  Removed.